### PR TITLE
AGDIGGER-125 Configure container cap option in K8s plugin

### PR DESCRIPTION
**Pull request by aliok**
How to verify:

```
ansible-playbook -i cluster-up-example sample-build-playbook.yml 
```

You might have problems with SSL (if really using local cluster). In that case, either add SSL cert of to Java's trusted certs or modify the `java -jar jenkins.cli.jar ...` in the playbook to use `-noCertificateCheck` flag.

You might also have "ERROR: anonymous is missing the Overall/Read permission" error. In that case change your private key with a no-password-protected one temporarily or modify the `java -jar jenkins.cli.jar ...` in the playbook to use `-i <new no-password private key>` flag.

### CODE REVIEW

File: configure-buildfarm/tasks/main.yml [hunk 1]
```diff
@@ -101,17 +101,17 @@
      - config
 
 -
-  name: "Create podtemplate update Groovy script from template"
+  name: "Create Kubernetes plugin config update Groovy script from template"
   template:
-    src: podtemplate-config.j2
-    dest: "/tmp/podtemplate-config.groovy"
+    src: kubernetes-plugin-config.j2
```
Comment (PhilipGough) [inline configure-buildfarm/tasks/main.yml]: I dont see this template renamed in this PR?

**Comment by PhilipGough** [comment]
@aliok this looks good with one comment I'm unsure about

File: configure-buildfarm/tasks/main.yml (hunk 1 above)
Comment (aliok) [inline configure-buildfarm/tasks/main.yml]: @PhilipGough https://github.com/aerogear/digger-installer/pull/25/files#diff-693f9f51e67aeff3fa75ed21a8e91fda

Comment (aliok) [inline configure-buildfarm/tasks/main.yml]: It is renamed. I am now thinking if there is another thing that I need to rename.

Comment (PhilipGough) [inline configure-buildfarm/tasks/main.yml]: No your correct, I missed that. Thanks for clarifying

**Pull request merged**